WorkaroundIncorrectSatelliteDeployment target can probably be removed from workarounds

The `WorkaroundIncorrectSatelliteDeployment` has a comment on it:

> Work around bug in Microsoft.NET.Sdk < v2.0 where satellites were deployed on top of each other in root. https://github.com/dotnet/sdk/issues/1360

The SDK bug has been fixed, so this target can probably be removed.  The workaround is causing issues when trying to pack a project:

> C:\git\dotnet-sdk\artifacts\.dotnet\2.1.300-preview3-008616\sdk\2.1.300-preview3-008616\Sdks\NuGet.Build.Tasks.Pack\build\NuGet.Build.Tasks.Pack.targets(198,5): error NU5118: File 'C:\git\dotnet-sdk\artifacts\Debug\bin\Tests\Microsoft.NET.Build.Tests\publish\de\Microsoft.DotNet.Cli.Utils.resources.dll' is not added because the package already contains file 'tools\netcoreapp2.1\any\de\Microsoft.DotNet.Cli.Utils.resources.dll' [C:\git\dotnet-sdk\src\Tests\Microsoft.NET.Build.Tests\Microsoft.NET.Build.Tests.csproj]

--- a/sdks/RepoToolset/tools/Imports.targets
+++ b/sdks/RepoToolset/tools/Imports.targets
@@ -6,7 +6,7 @@
     <MSBuildAllProjects>$(MSBuildAllProjects);$(MSBuildThisFileFullPath)</MSBuildAllProjects>
   </PropertyGroup>
 
-  <Import Project="Sdk.targets" Sdk="Microsoft.NET.Sdk" />
+  <Import Project="Sdk.targets" Sdk="Microsoft.NET.Sdk" Condition="'$(ImportNetSdkFromRepoToolset)' == 'true'" />
   
   <PropertyGroup>
     <DeployProjectOutput Condition="'$(DeployProjectOutput)' == ''">$(__DeployProjectOutput)</DeployProjectOutput>

--- a/sdks/RepoToolset/tools/Settings.props
+++ b/sdks/RepoToolset/tools/Settings.props
@@ -4,6 +4,8 @@
 
   <PropertyGroup>
     <MSBuildAllProjects>$(MSBuildAllProjects);$(MSBuildThisFileFullPath)</MSBuildAllProjects>
+
+    <ImportNetSdkFromRepoToolset Condition="'$(ImportNetSdkFromRepoToolset)' == ''">true</ImportNetSdkFromRepoToolset>
   </PropertyGroup>
 
   <Import Project="BuildTasks.props" />
@@ -21,5 +23,5 @@
   <Import Project="VisualStudio.props" Condition="'$(UsingToolVSSDK)' == 'true'"/>
 
   <!-- Directory.Build.props are imported next by .NET SDK (via Common.targets) -->
-  <Import Project="Sdk.props" Sdk="Microsoft.NET.Sdk" />
+  <Import Project="Sdk.props" Sdk="Microsoft.NET.Sdk" Condition="'$(ImportNetSdkFromRepoToolset)' == 'true'" />
 </Project>

--- a/sdks/RepoToolset/tools/Workarounds.targets
+++ b/sdks/RepoToolset/tools/Workarounds.targets
@@ -36,19 +36,6 @@
   </Target>
 
   <!--
-    Work around bug in Microsoft.NET.Sdk < v2.0 where satellites were deployed on top of each other in root.
-    https://github.com/dotnet/sdk/issues/1360
-  -->
-  <Target Name="WorkaroundIncorrectSatelliteDeployment" AfterTargets="ResolveLockFileCopyLocalProjectDeps">
-    <ItemGroup>
-      <ReferenceCopyLocalPaths Remove="@(ResourceCopyLocalItems)" />
-      <ReferenceCopyLocalPaths Include="@(ResourceCopyLocalItems)"  Condition="'@(ResourceCopyLocalItems)' != ''">
-        <DestinationSubDirectory>$([System.IO.Directory]::GetParent(%(ResourceCopyLocalItems.FullPath)).get_Name())\</DestinationSubDirectory>
-      </ReferenceCopyLocalPaths>
-    </ItemGroup>
-  </Target>
-
-  <!--
     XAML targets create a temp project with OutDir set, which makes the SDK create an empty directory for it,
     polluting the output dir. Avoid creating these directories.
     https://github.com/dotnet/sdk/issues/1367

